.github/workflows/workstatus-creation.yaml

```diff
@@ -1,4 +1,4 @@
-name: test workstatus creation
+name: test workstatus operations
```

Comment:
So I would change the name of this file too.

Reply:
Thanks! Did it by #29 .